approval_polls/templates/approval_polls/detail.html

```diff
@@ -94,6 +94,20 @@
 
 <div class='row-fluid top-buffer'>
   <div class='col-md-12'>
+    <p>Poll Tags:
+    {% if num_tags == 0 %}
+      None for this poll
```

Comment:
Perhaps you could skip displaying anything if no tags are available ? Also it should be sufficient with the label `Tags:` alone. Just wanted the keep the text to minimum since it may be embedded in an iframe.